Bitrise/testing_bitrise.yml

```diff
@@ -80,6 +80,14 @@ workflows:
                 cd Submodules/WeTransfer-iOS-CI
                 swift run danger-swift ci --cache-path ../../.build --build-path ../../.build --cwd ../../
             fi
+    - deploy-to-bitrise-io@2:
+        is_skippable: true
```

Comment:
Why did we decide to make this skippable? If we don't have results, shouldn't something have gone wrong? 😛

Reply:
Some PRs don't have tests to run, which means there's no `xcresult` to upload. I couldn't comment inline, I would've done that otherwise. Hope this explanation helps!